tests/foreman/ui/test_user.py

```diff
@@ -249,11 +247,25 @@ def test_positive_create_user_different_pass(self, password):
         @Assert: User is created
 
         """
-        name = gen_string('alpha')
+        test_data = valid_strings()
```

Comment:
Lines 250-257 can be very well combined in to the for statement in Line 259 but it makes the loop look so complex.  So I split it for readability.